src/platform/windows/drop_handler.rs

```diff
@@ -81,12 +83,13 @@ impl FileDropHandler {
     ) -> HRESULT {
         use events::WindowEvent::HoveredFile;
         let drop_handler = Self::from_interface(this);
-        Self::iterate_filenames(pDataObj, |filename| {
+        let hdrop = Self::iterate_filenames(pDataObj, |filename| {
             send_event(Event::WindowEvent {
                 window_id: SuperWindowId(WindowId(drop_handler.window)),
                 event: HoveredFile(filename),
             });
         });
+        drop_handler.hovered_is_valid = hdrop.is_some();
```

Comment:
When you drag an unsupported format over the Winit window, the cursor still turns into the valid drop cursor. Could it be set to the invalid drop cursor? I think `pdwEffect` may be what Windows uses to support that.